zwave_js_server/client.py

```diff
@@ -165,7 +165,7 @@ async def listen(self, driver_ready: asyncio.Event) -> None:
 
             state_msg = await self._receive_json_or_raise()
 
-            if not state_msg["success"]:
+            if not state_msg.get("success"):
```

Comment:
When is the success key missing?

Reply:
it is related to https://github.com/home-assistant/core/issues/47347

Reply:
If the success key is missing it means we're not receiving the expected message type from the server. We need to look into why this is.

Reply:
This is a side effect of sending setValue commands to battery powered nodes (which we no longer await in the python lib). I think we should fix this in the server and not send back an (unexpected) result for commands that we didn't want to await.

Reply:
I think the wrong issue was linked, it's https://github.com/home-assistant/core/issues/47357.

I added some logs to that, in my case it had nothing to do with setValue or battery nodes, just a value update message arriving at an unexpected time.

Reply:
ah ok, in that case we could it as-is for now.